resolwe/flow/executors/docker/run.py

```diff
@@ -32,6 +33,52 @@
 logger = logging.getLogger(__name__)
 
 
+def retry(
```

Comment:
We use (almost) the same decorator in storage manager. It would be nice to combine them if possible.

Reply:
Executor virtual environment does not have storage manager code available. I could use the one from the docker executor on both ends.